client/src/providers/HealthProvider.tsx

```diff
@@ -133,6 +136,60 @@ export const HealthProvider: React.FC<Props> = ({ children, ...props }) => {
     }
   };
 
+  const requestMedicalRecordHealthBlockContract = async (patientAddress: string) => {
```

Comment:
remove console.log statements and try returning the info in the method to validate

Reply:
removed console.log statements